notes/lecture25.tex

```diff
@@ -0,0 +1,182 @@
+\begin{equation}
+\bar{x}=x-[\nabla^2f_\epsilon(x)]^{-1}\nabla f_\epsilon(x)=x-[\epsilon A^\trans S^{-2}A]^{-1}\left(c-\epsilon A^\trans S^{-1}\mathbb{1}\right)\label{eq:Newton_update}
+\end{equation}
+
+Recall that Newton's method finds the solution by making the first-order condition zero. To measure how much the Newton update will decrease the first-order approximation, we introduce the concept of Newton decrement.
+
+Define the Newton decrement as
+\begin{equation}
+q^2(x, \epsilon) = \nabla f^\trans_\epsilon(x)[\nabla^2f_\epsilon(x)]^{-1}\nabla f_\epsilon(x)\label{eq:newton_decrement}
+\end{equation}
+or $q(x, \epsilon) = \left\|[\nabla^2f_\epsilon(x)]^{-1/2}\nabla f_\epsilon(x)\right\|_2$. 
+
+Note that the Newton decrement also relates to the difference between $f_\epsilon(x)$ and the minimum of its second-order approximation:
+\begin{eqnarray}
+&&f_\epsilon(x)-\min_{\bar{x}}\left(f_\epsilon(x)+\nabla f_\epsilon(x)^\trans(\bar{x}-x)+(\bar{x}-x)^\trans\nabla^2 f_\epsilon(x)(\bar{x}-x)\right)\nonumber\\
+&=&f_\epsilon(x)-\left(f_\epsilon(x)-\dfrac{1}{2}\nabla f^\trans_\epsilon(x)[\nabla^2f_\epsilon(x)]^{-1}\nabla f_\epsilon(x)\right)\nonumber\\
+&=&\dfrac{1}{2}\nabla f^\trans_\epsilon(x)[\nabla^2f_\epsilon(x)]^{-1}\nabla f_\epsilon(x)\triangleq \dfrac{1}{2}q^2(x, \epsilon).
+\end{eqnarray}
+
+We'll then use Newton decrement to find out the conditions for convergence guarantee of the algorithm. 
+
+\subsubsection{Algorithm with Convergence Guarantee}
+We'll now come up with an update rule that can guarantee convergence if some initial conditions are satisfied. To develop the update rule, we first introduce the following propositions. 
+\begin{proposition}\label{prop1}
+Assume $Ax > b$ and $q(x, \epsilon)<1$, then we have
+\begin{align}
+c^\trans x-c^\trans x^{*}\leq 2\epsilon n.
+\end{align}
+\end{proposition}
+In particular, if we maintain that $x_t$ is interior point satisfying $Ax_t>b$,
+and $q(x_t, \epsilon_t) < 1$, then $c^\trans x_t$ converges to $c^\trans x^*$ as $\epsilon_t$ goes to $0$, i.e., $x_t$ converges to global optimum. However, the condition $q(x_t,\epsilon_t)<1$ is not trivial.  
+
+\begin{proposition}\label{prop2}
+If $Ax>b$, and $q(x,\epsilon)<1$, then the pure Newton iterate
+step $\bar{x}$ satisfies,
+\begin{align}
+q(\bar{x}, \epsilon) &\leq q(x, \epsilon)^2 
+\end{align}
+\end{proposition}
+It ensures that $q(\bar{x}, {\epsilon})<1$ given $q(x,\epsilon)<1$ and $x$ is interior point. But we also want that $q(\bar{x}, \bar{\epsilon})<1$ for some $\bar{\epsilon}<\epsilon$. 
+
+\begin{proposition}\label{prop3}
+Assume $q(x,\epsilon)\leq \frac{1}{2}$, interior point $Ax>b$,
+put 
+\begin{align}
+\bar{\epsilon} = \left(1-\frac{1}{6\sqrt{n}}\right)\epsilon,
+\end{align}
+then we have
+\begin{align}
+q(\bar{x}, \bar{\epsilon})\leq \frac{1}{2}
+\end{align}
+\end{proposition}
+These propositions suggest the following update rule,
+\begin{align}
+x_{t+1} & = x_t - \nabla^\trans f_{\epsilon_t}(x)^{-1}\nabla f_{\epsilon_t}(x_t) \\
```

Comment:
No \trans afte \nabla. Was this perhaps a search/replace issue? Worth double checking.